packages/core/src/node/build.ts

```diff
@@ -129,10 +138,14 @@ export async function renderPages(
             try {
               ({ appHtml } = await render(routePath, helmetContext.context));
             } catch (e) {
+              if (typeof ssgConfig === 'object' && ssgConfig.strict) {
```

Comment:
Do you think we should provide additional context to assist users with debugging? like add `routePath` or more to the error.message

Reply:
Sure 👍  the error logs has been updated:

<img width="1127" alt="Screenshot 2024-08-14 at 11 48 44" src="https://github.com/user-attachments/assets/1d8b3a25-aadb-4a5f-9c3a-c837acf563c0">

Reply:
Cool. Really helpful error diagnostic